admin_guide/assembly_restoring-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+[[admin-guide-restore]]
+= Restoring your cluster
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: admin-restore-cluster
+
+toc::[]
+
+== Overview
+
+In {product-title}, you can _restore_, recreating state from separate storage,
+at the cluster level.
+
+To restore a cluster, you must first xref:../day_two_guide/environment_backup.adoc#day_two_environment_backup[back it up].
+
```

Comment:
This links to "Creating an environment-wide backup". Should the link include that wording: _back up your environment_ ? (I find it a touch confusing when the linked-to information is differently worded.)

Reply:
I think it's ok.
- While I can't remember the link offhand, I've seen in one of our guidelines that we're not supposed to use the "See [page title]" construction and instead make links flow into text. I think that putting the page title in the forbidden construction makes a lot of sense, but it makes less sense to put the page title in a sentence.
- The [examples we have in the readme](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-internal-topics) don't use page titles.

What do you find confusing? Should we consider changing our approach?

Reply:
Just that the link text is different than the heading the link goes to. Took me a moment to make sure I was in the the right place. Maybe just me...